src/Campaigns/Actions/CreateDefaultLayoutForCampaignPage.php

```diff
@@ -25,12 +23,12 @@ class CreateDefaultLayoutForCampaignPage
     /**
      * @unreleased
      */
-    public function __invoke(Campaign $campaign)
+    public function __invoke(array $campaign): string
     {
         $layout = array_map(function($block) use ($campaign) {
             return str_replace(
                 ['%id%', '%description%'],
-                [$campaign->id, $campaign->shortDescription],
+                [$campaign['id'], $campaign['shortDescription']],
```

Comment:
Why don't we just pass in the `$campaignId` and `$shortDescription` as params instead of an array?  I'm not sure our future selves will remember what the specific array shape will be.

Reply:
Thanks!